src/spdx/clitools/pyspdxtools.py

```diff
@@ -72,7 +79,20 @@ def main(infile: str, outfile: str, version: str, novalidation: bool):
             else:
                 logging.info("The document is valid.")
 
-        if outfile and outfile != "-":
+        if outfile == "-":
+            tagvalue_writer.write_document(document, sys.stdout)
+
+        elif graph:
+            try:
+                export_graph_from_document(document, outfile)
```

Comment:
Maybe insert a catch that `outfile` must be set, i.e. not `None`, and inform the user about it?

Reply:
sounds like a nice to have. Would merge it now